packages/plugin-rsc/e2e/starter.test.ts

```diff
@@ -245,6 +245,112 @@ test.describe(() => {
   })
 })
 
+test.describe(() => {
+  const root = 'examples/e2e/temp/render-built-url-runtime'
+
+  test.beforeAll(async () => {
+    await setupInlineFixture({
+      src: 'examples/starter',
+      dest: root,
+      files: {
+        'vite.config.ts': /* js */ `
+          import rsc from '@vitejs/plugin-rsc'
+          import react from '@vitejs/plugin-react'
+          import { defineConfig } from 'vite'
+
+          export default defineConfig({
+            plugins: [
+              react(),
+              rsc({
+                entries: {
+                  client: './src/framework/entry.browser.tsx',
+                  ssr: './src/framework/entry.ssr.tsx',
+                  rsc: './src/framework/entry.rsc.tsx',
+                }
+              }),
+            ],
+            experimental: {
+              renderBuiltUrl(filename) {
+                return {
+                  runtime: \`'/' + \${JSON.stringify(filename)}\`
+                }
+              }
+            }
```

Comment:
It would be nice if you can setup test like how you actually use. Can you try testing `globalThis.__assetsPath` as an example?

Reply:
Yeah I was thinking about it. Should we rewrite the enter.ssr/rsc.tsx to add this global or is there a better place?

Reply:
For build/preview time, we can probably use `configurePreviewServer` to simulate `globalThis` override and also custom asset prefix. I added an example in "renderBuiltUrl-string" test below.

Reply:
Or you can also override the entry by:

```ts
setupInlineFixture({
  src: 'examples/starter',
  dest: root,
  files: {
    'vite.config.ts': /* js */ `
      ...
      export default defineConfig({
        plugins: [
          rsc({ entries: { ..., rsc: "./src/framework/entry.rsc.custom.tsx" } }
        ]
      })
    `,
    'src/framework/entry.custom.tsx': `
       export { handler } from "./entry.rsc.tsx"
       globalThis.__assetsUrl = () => ....
     `
  },
})
```

Do you have some pattern in mind how/where you actually define `globalThis`?